### ISSUE TITLE
There is a runner

### ISSUE THREAD

**Issue by Kazamaru** [EnigmaDragons/BreakTheIce#2]

**Comment by ZavixDragon** [comment]
is this an art ticket as in you want art for a runner or something else?

**Comment by Kazamaru** [comment]
No, I literally just want a runner to exist. I believe this falls under backend. Do you think I should first make a ticket for content creation where I give the runner a name and then make a ticket for making that runner exist in the backend?

**Comment by SilasReinagel** [comment]
Naming a runner can be another ticket. 

Having a UI view where the player can input the name, can be yet another ticket.

**Comment by ZavixDragon** [comment]
Also I don't know why we make a backend runner until we have a feature that needs it
